web/gds-user-ui/src/components/Section/NetworkActivity.tsx

```diff
@@ -0,0 +1,57 @@
+import { VStack } from '@chakra-ui/react';
```

Comment:
Here's a link to the Victory docs: https://formidable.com/open-source/victory/docs